### PR TITLE
chore(appearance): creating new component for state ui testing (VIV-000)

### PR DESCRIPTION
**Pull request by rachelbt**

### CODE REVIEW

File: apps/docs/_data/components.json [hunk 1]
```diff
@@ -289,5 +289,10 @@
 		"title": "audio-player",
 		"status": "alpha",
 		"markdown": "./libs/components/src/lib/audio-player/README.md"
+	},
+	{
```
Comment (TaylorJ76) [inline apps/docs/_data/components.json:293]: Can this be removed in favour of "Appearance" in `designs.json`?

File: docs/design/appearance.md [hunk 2]
```diff
@@ -0,0 +1,312 @@
...
+
+---
+
+Vivid appearance is setting the way the component looks (ghost, outline, filled, etc).  
+Appearance is also connected to connotation and setting colors to each state in the appearance.
+
+---
+## How does it work?
+Each component is using at least one appearance.  
+The appearance control the colors that are set for text, fill and outline.
```
Comment (TaylorJ76) [inline docs/design/appearance.md:11]: control**s**

File: docs/design/appearance.md [hunk 3]
```diff
@@ -0,0 +1,312 @@
...
+---
+
+Vivid appearance is setting the way the component looks (ghost, outline, filled, etc).  
+Appearance is also connected to connotation and setting colors to each state in the appearance.
+
+---
+## How does it work?
+Each component is using at least one appearance.  
+The appearance control the colors that are set for text, fill and outline.  
+The 3 colors are changing according to stats (idle, hover, disabled, etc).
```
Comment (TaylorJ76) [inline docs/design/appearance.md:12]: stat**e**s

**Review by TaylorJ76 — CHANGES_REQUESTED** [review]

File: libs/components/src/lib/appearance-ui/README.md
[the remark this answers is not in the document]
Comment (rachelbt) [reply libs/components/src/lib/appearance-ui/README.md:1]: I want to have this for the ui tests.
I guess I can copy-past the examples to the ui-test

File: docs/design/appearance.md (hunk 3 above)
Comment (rachelbt) [reply docs/design/appearance.md:12]: :woman-facepalming::skin-tone-2:

File: libs/components/src/lib/appearance-ui/README.md
[the remark this answers is not in the document]
Comment (RichardHelm) [reply libs/components/src/lib/appearance-ui/README.md:1]: Can't you just take them the from the other file?
```js
path.join(new URL('.', import.meta.url).pathname, '../../../../../docs/design/appearance.md')
```

**Review by TaylorJ76 — APPROVED** [review]

File: libs/components/src/lib/appearance-ui/README.md
[the remark this answers is not in the document]
Comment (rachelbt) [reply libs/components/src/lib/appearance-ui/README.md:1]: I'll try :)

**Review by TaylorJ76 — APPROVED** [review]

**Pull request merged**